spec/support/mockers/equipment_model.rb

```diff
@@ -0,0 +1,31 @@
+require Rails.root.join('spec/support/mockers/mocker.rb')
+require Rails.root.join('spec/support/mockers/category.rb')
+require Rails.root.join('spec/support/mockers/equipment_item.rb')
+
+class EquipmentModelMock < Mocker
```

Comment:
This is AWESOME :smile:. The usage for a Reservation mocker in #1582 was kinda nifty, but this is way slicker and the benefit of the `Mocker` superclass is much easier to see.